Unittest.py

```diff
@@ -9,8 +9,10 @@
 
 from ItemList import item_table
 from ItemPool import remove_junk_items, item_groups
+from Location import LocationIterator
 from LocationList import location_groups, location_is_viewable
 from Main import main, resolve_settings, build_world_graphs
+from Plandomizer import is_pattern, pattern_matcher
```

Comment:
Oh no, this is going to conflict with #1233's move of pattern_matcher since it now needs the Distribution object. Can we still retrieve that out of the spoiler object perhaps?